### DISCUSSION TITLE
Content "Missing or empty"

### DISCUSSION THREAD

**Discussion by zechs-merquise** [tailwindlabs/tailwindcss#8427] [Help]
I am installing tailwind and this error happens. Followed every step the correct way. Can someone help me?
Thanks!
![a](https://user-images.githubusercontent.com/106116790/170125726-1a99005d-f53e-439a-8253-6f2bd168c5a9.JPG)
![b](https://user-images.githubusercontent.com/106116790/170125727-13d425c9-ce92-4941-9c5f-4a8408ddfd43.JPG)

**Comment by adamwathan** [comment]
Hey! Can you provide a reproduction? Definitely shouldn’t be seeing that with your config 🤔

**Reply by zechs-merquise** [reply]
I'm trying to install Tailwind, and this is what happens.
![c](https://user-images.githubusercontent.com/106116790/170142777-92176fdc-87e6-4f0e-942d-ebdc7e86a0cb.JPG)
![d](https://user-images.githubusercontent.com/106116790/170142782-6924ef28-402d-47a4-aa91-5cb8ef96e094.JPG)
![e](https://user-images.githubusercontent.com/106116790/170142786-152d300d-13bf-4971-8578-d28ef807c37a.JPG)

**Reply by zechs-merquise** [reply]
When I type npm run dev, that error appears

**Reply by zechs-merquise** [reply]
![image](https://user-images.githubusercontent.com/106116790/170148145-c8314026-698d-47c0-86de-fe91b0b5c00f.png)

**Comment by zechs-merquise** [comment]
![image](https://user-images.githubusercontent.com/106116790/170306481-af208efb-ca14-4ce2-8cf4-f2f923ca21de.png)
I think the directives are causing the problem.

**Comment by grety22** [comment]
@zechs-merquise for the issue : ### **The `content` option in your Tailwind CSS configuration is missing or empty**
Just replace double quotes for single quotes for each element inside your content array, and recompile (npm run dev)
![image](https://user-images.githubusercontent.com/23240307/173272787-5db314c3-ab71-4ee6-8515-311acc3881b9.png)

**Comment by grety22** [comment]
For those issues in your App component you have to fix the arrow function and also inside your return you must add a wrapper element since you cant have sibblings (div and h1) without a parent, if you dont want to have extra div, you can use Fragment tag, either like `<Fragment><your div><your h1></Fragment`> or in a shorthand tag for fragment : `<><your div><your h1></>` 
![image](https://user-images.githubusercontent.com/23240307/173273906-ed3f42c2-f90e-4efe-ac5c-3ac04dd85228.png)
